### PR TITLE
Feature/add multinamespace multicluster

### PR DESCRIPTION
**Pull request by antonioberben**
This PR:

In convert:
- Updates convert to allow topologies in multiple namespaces and multiple clusters
- Adds following attributes to convert: 
>      --client-disabled                             disabling client service (fortio) as part of the output
>      --client-image string                         the image to use for the load testing client job (default "fortio/fortio")
>      --client-namespace string                     namespace where to deploy the client (default "default")
>      --cluster string                              the cluster name to generate related workloads. It needs to match the cluster attribute for the services
>      --service-image string                        the image to deploy for all services in the graph

- Removes Graphviz
- Updates tests to cover new features
- Removes deprecated RBACPolicies
- Adds option to add the client (fortio) in a different namespace or even to not be part of the output services
- Adds example of topologies
- Adds documentation in README

In isotope service:
- Adds CONFIG_PATH as environment variable for the topology config file
- Adds documentation in README
- Adds the possibility to include headers to call topology attribute (in example when a service in the middle requires a header to pass an authorization mechanism only applied to that service)

### CODE REVIEW

File: isotope/README.md [hunk 1]
```diff
@@ -71,6 +68,11 @@ services:
   - call:
       service: b
       payloadSize: 80B
+  - call:
+      service: b
+      call-override: b:8080 # Override the URI used in the call (i.e. when the service is deployed in a different namespace and it requires to specify the namespace in the hostname: `b.namespace.svc.cluster.local:8080`)
```
Comment (howardjohn) [inline isotope/README.md:73]: nit: `hostname` feels a bit more explicit than `call-override`

File: isotope/convert/pkg/graph/svc/service.go [hunk 2]
```diff
@@ -26,6 +26,12 @@ type Service struct {
 	// Name is the DNS-addressable name of the service.
 	Name string `json:"name"`
 
+	// Name is the DNS-addressable name of the service.
```
Comment (howardjohn) [inline isotope/convert/pkg/graph/svc/service.go:29]: comment here and below is wrong

**Review by howardjohn — APPROVED** [review]

File: isotope/README.md (hunk 1 above)
Comment (antonioberben) [reply isotope/README.md]: fixed

File: isotope/convert/pkg/graph/svc/service.go (hunk 2 above)
Comment (antonioberben) [reply isotope/convert/pkg/graph/svc/service.go]: fixed

**Comment by stevenctl** [comment]
Is there a doc or issue describing the motivation and plan for this?

**Comment by antonioberben** [comment]
Hi @stevenctl , happy to create an issue. But I think it is not activated in this repo

**Comment by stevenctl** [comment]
Just wondering if there's a multi cluster testing effort within the project that I wasn't aware of.

**Comment by antonioberben** [comment]
@stevenctl , I am not aware either. But, just adding that cluster flag to the tool, you can test topologies cross cluster.

Example: You run the [multi-cluster istio setup](https://istio.io/latest/docs/setup/install/multicluster/verify/). But instead of using the `helloworld.sample`, you want to use `helloworld.global`. Until you do not create the required resources (i.e. `ServiceEntry`) the cross cluster communication will not work.

Then, with isotope and the new flags, you can deploy topologies and develop which Istio resources are required for your needs. Like cross cluster.

This is a use-case I am writing about. I use a fork of this project. But it felt better if the feature was part of the original project. Then, this PR.

Personally, I wish this project had more audience because I found many times people trying to create mock services for demos or to test scenarios out, but using something ad-hoc. This tool, besides testing performance, it can also help to improve the developer experience ("Me, as a user, I am not sure how to setup failover. then, I define a cros cluster topology with isotope, and test my istio resources to prove that the failover happens before using an actual application.")

**Pull request merged**